src/registry/routes/plugins.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = function(conf) {
+  return function(req, res) {
+    if (conf.discovery) {
+      const plugins = Object.entries(conf.plugins).map(
+        ([pluginName, pluginFn]) => ({
+          name: pluginName,
+          description: pluginFn.toString()
```

Comment:
I'm not too fond of us abusing `toString` on plugin function for description. I'd rather we had a property `description` we could refer to

Reply:
so abusing properties in functions it's fine then? :P